WordPress/Sniff.php

```diff
@@ -550,6 +550,76 @@ public function strip_quotes( $string ) {
 	}
 
 	/**
+	 * Add a PHPCS message to the output stack as either a warning or an error.
+	 *
+	 * @since 0.11.0
+	 *
+	 * @param string $message   The message.
+	 * @param int    $stackPtr  The position of the token the message relates to.
+	 * @param bool   $is_error  Optional. Whether to report the message as an 'error' or 'warning'.
+	 *                          Defaults to true (error).
+	 * @param string $code      Optional error code for the message. Defaults to 'Found'.
+	 * @param array  $data      Optional input for the data replacements.
+	 * @param int    $severity  Optional. Severity level. Defaults to 0 which will translate to
+	 *                          the PHPCS default severity level.
+	 * @return bool
+	 */
+	protected function addMessage( $message, $stackPtr, $is_error = true, $code = 'Found', $data = array(), $severity = 0 ) {
```

Comment:
Over and above wordy variables, I suppose the larger problem is positional arguments as opposed to passing associative arguments. Compare:

```php
$this->addMessage( 'Hello World', $stackPtr, true, 'Found', array(), 0 );
```

vs:

```php
$this->addMessage( array(
    'message' => 'Hello World', 
    'stack_pointer' => $stackPtr,
    'is_error' => true, 
    'code' => 'Found',
    'data' = array(), 
    'severity' => 0 
) );
```

Sure this is more verbose, but each of the params then become optional whereas with positional arguments they are all required up to the last argument you need to supply.

Anyway, I don't feel strongly about this, but I did want to offer it up as a suggestion.